static/app/utils/performance/suspectSpans/suspectSpansQuery.tsx

```diff
@@ -38,7 +40,12 @@ function getSuspectSpanPayload(props: RequestProps) {
     delete payload.spanGroup;
   }
   const additionalPayload = props.eventView.getEventsAPIPayload(props.location);
-  return Object.assign(payload, additionalPayload);
+  return {
+    ...payload,
+    ...additionalPayload,
+    min_exclusive_time: minExclusiveTime,
+    max_exclusive_time: maxExclusiveTime,
```

Comment:
Nit: I think `min_exclusive_time`, `max_exclusive_time` would fit better being in the `payload` object

Reply:
done!